WordPress/Classes/Extensions/UIImageView+SiteIcon.swift

```diff
@@ -66,13 +66,19 @@ extension UIImageView {
             return
         }
 
-        if !blog.isPrivate() {
-            setImageWith(siteIconURL, placeholderImage: placeholderImage)
-            return
+        let request: URLRequest
+        if blog.isPrivate() {
+            request = PrivateSiteURLProtocol.requestForPrivateSite(from: siteIconURL)
+        } else {
+            request = URLRequest(url: siteIconURL)
         }
 
-        let request = PrivateSiteURLProtocol.requestForPrivateSite(from: siteIconURL)
-        setImageWith(request, placeholderImage: placeholderImage, success: nil, failure: nil)
+        setImageWith(request, placeholderImage: placeholderImage, success: { [weak self] (_, _, image) in
+            self?.image = image
+            self?.removePlaceholderBorder()
+        }) { [weak self] (_, _, _) in
```

Comment:
Nitpick, but could we keep the name of the final closure instead of using trailing closure syntax in this case? I think it's a little clearer.